examples/aggregator/scripts/deploy

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env bash
 
-readonly GLOBAL_SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")/../../../scripts/"
 readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+readonly GLOBAL_ROOT_DIR="${SCRIPTS_DIR}/../../.."
+readonly GLOBAL_SCRIPTS_DIR="${GLOBAL_ROOT_DIR}/scripts/"
 # shellcheck source=scripts/common
 source "${GLOBAL_SCRIPTS_DIR}/common"
 
 "${SCRIPTS_DIR}/undeploy" || true
 
 gcloud container clusters get-credentials aggregator
-kubectl apply --filename=examples/aggregator/gcp/pod.yaml
+kubectl create secret generic tls-private-key \
+  --from-file="${GLOBAL_ROOT_DIR}/examples/certs/docker/docker.key"
```

Comment:
We should rename this key to something like "gcp" rather than "docker", at some point?

Reply:
Created #826 to do this.